### PR TITLE
[PyCDE] Pass module parameters as generate function args

### PR DESCRIPTION
**Pull request by teqdruid**
We need the ability to convert `Attribute`s back to python values for PyCDE. This conversation [1], however, isn't going anywhere fast. This is a hack to get us unblocked until that conversation resolves.

[1] https://llvm.discourse.group/t/python-attribute-concreteattribute/3563

### CODE REVIEW

File: frontends/PyCDE/src/pycde/module.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #  See https://llvm.org/LICENSE.txt for license information.
 #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from __future__ import annotations
```
Comment (mikeurbach) [inline frontends/PyCDE/src/pycde/module.py:5]: Please keep this, it is needed to support the `dict[...]` and `list[...]` type hint syntax on Python 3.7 and 3.8.

File: frontends/PyCDE/src/pycde/module.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+from .hacks import attribute_to_var
```
Comment (mikeurbach) [inline frontends/PyCDE/src/pycde/module.py:5]: Do we really need to name the module "hacks"?

Comment (mikeurbach) [reply frontends/PyCDE/src/pycde/module.py:5]: IMO `attribute_to_var` should live in `support`, alongside `var_to_attribute`, even if the implementation is considered hacky.

**Review by mikeurbach — APPROVED** [review]
LGTM in general.

File: frontends/PyCDE/src/pycde/module.py (hunk 1 above)
Comment (teqdruid) [reply frontends/PyCDE/src/pycde/module.py:5]: I was wondering why that was there...

File: frontends/PyCDE/src/pycde/module.py (hunk 2 above)
Comment (teqdruid) [reply frontends/PyCDE/src/pycde/module.py:5]: Yeah, I thought about both. For the `hacks` name: I wanted to make it very clear that I consider this implementation to be _very_ hacky and should be replaced ASAP once the "correct" solution appeared. I chose to put it in pycde just to keep hacky stuff out of the bindings which none of the bindings code needs. (I'm thinking now that some of the stuff in `support.py` are kind of opinionated and should be moved into pycde.)

I don't feel strongly about either decision so if you have any opinion whatsoever, I'll move it.

**Comment by mikeurbach** [comment]
Thanks!

**Pull request merged**